### DISCUSSION TITLE
Separating "/cmi clear" command permissions.

### DISCUSSION THREAD

**Discussion by barannnnnns** [Zrips/CMI#7016] [Enhancement]
"cmi.command.clear" chat next to your leave the "cmi.command.clearchat.self" permission can be added.

Personally, I want players to be able to delete their own chats. However, there is currently no such distinction.

**Comment by natr566** [comment]
You could probly do that with a custom alias. by adding lots of blank line. As thats what clear chat basically does

**Reply by barannnnnns** [reply]
Yes I managed to do this with bperm(Specialized Commands). Thank you.

**Comment by mrfloris** [comment]
Clear = remove items from inventory
clearchat = remove chat from players
two different things :)

But here is a list of all the clear commands (clear enderchest, inventory, ground items, and chat msgs)
<img width="792" alt="Screen-Shot 2022-09-25 at 19 00 11" src="https://user-images.githubusercontent.com/28841349/192155761-bc936126-99c2-4180-82ed-fa4c92e2300c.png">

```
<img width="396" alt="Screen-Shot 2022-09-25 at 19 01 11" src="https://user-images.githubusercontent.com/28841349/192155806-b07339b9-8b8f-435c-b7f6-b79c0f404f7e.png">

/cmi clearchat 
removes the chat from all users, you as server admin will still see it all so you can manage the server

/cmi clearchat self
removes the chat from yourself, wont effect others online.

**Comment by provodd** [comment]
It's a pity that:
There is no way to grant players permission to clear only their own chat instead of the entire chat.
There is no way to clear a specific player's chat via the console.

Would it be possible to add these features in future updates?
